tools/syntax.txt

```diff
@@ -982,6 +985,24 @@ list<SyntaxNode> items
 token end
 NamedBlockClauseSyntax endBlockName
 
+LetPortSyntax kind=LetPort
+list<AttributeInstanceSyntax> attributes
+DataTypeSyntax type
+token identifier
```

Comment:
You could probably reuse VariableDeclaratorSyntax for the following three entries and save some work.